### PR TITLE
Add us_ice crawler

### PR DESCRIPTION
**Pull request by atehe**
Closes [US Immigration and Customs Enforcement #32](https://github.com/opensanctions/crawler-planning/issues/32)

### CODE REVIEW

File: datasets/us/ice_wanted/crawler.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        xpath_value (str):  xpath to extract text from
+        to_remove (list, optional): string to remove in the extracted text.
+    """
+    element_tags = doc.xpath(xpath_value)
+
+    tag_list = []
+    for tag in element_tags:
+        tag_list.append(tag.text_content())
+
+    element_text = "".join(tag_list)
```
Comment (jbothma) [inline datasets/us/ice_wanted/crawler.py:20]: let's join on `" "` so we don't inadvertently get something like `end of something.New something` - or is that prevented in some other way?

File: datasets/us/ice_wanted/crawler.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    for string in to_remove:
+        element_text = element_text.replace(string, "")
+
+    return collapse_spaces(element_text.strip())
+
+
+def generate_mw_notes(**kwargs):
+    note_string = f"""Additional Information:
+- Wanted For: {kwargs.get('crime_summary')}
+- Wanted Heading: {kwargs.get('crime_details')}
```
Comment (jbothma) [inline datasets/us/ice_wanted/crawler.py:31]: we don't really define or render this as markdown and we don't render newlines on the website. What could work nicely is if this was a list of strings without the dash instead. Then each attribute would be separated and easy to render in reasonable ways for everyone.

File: datasets/us/ice_wanted/crawler.py [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    doc = context.fetch_html(url, cache_days=1)
+
+    names = get_element_text(
+        doc,
+        '//div[contains(@class, "field--name-field-most-wanted-name")]//div[contains(@class, "field__item")]',
+    )
+    names_split = names.split(",")
+
+    name = names_split[0]
+    alias = names_split[1:]
```
Comment (jbothma) [inline datasets/us/ice_wanted/crawler.py:55]: as far as I can tell, the name is actually `surname(s), forename(s)` rather tan `name, aliases`. Or do you have different examples?

There's an alias field which I think we're missing right now.

File: datasets/us/ice_wanted/us_ice_wanted.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+title: "U.S. Immigration and Customs Enforcement Most Wanted Fugitives"
+entry_point: crawler.py
+prefix: us-ice-want
```
Comment (jbothma) [inline datasets/us/ice_wanted/us_ice_wanted.yml:3]:
```suggestion
prefix: us-icew
```

File: datasets/us/ice_wanted/crawler.py [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+- Wanted Heading: {kwargs.get('crime_details')}
+- Wanted Category: {kwargs.get("category")}
+- Wanted Status: {kwargs.get("status")}
+- Age: {kwargs.get("age")}
+- Height: {kwargs.get("height")}
+- Weight: {kwargs.get("weight")}
+- Skin tone: {kwargs.get("skin_tone")}
+- Eyes: {kwargs.get("eyes")}
+- Hair: {kwargs.get("hair")}
+- Last Known Location: {kwargs.get('last_known_location')}"""
```
Comment (jbothma) [inline datasets/us/ice_wanted/crawler.py:40]: i think you can stuff this into `Person:address`

File: datasets/us/ice_wanted/crawler.py (hunk 2 above)
Comment (jbothma) [reply datasets/us/ice_wanted/crawler.py:31]: and the descriptive attributes in the description property, while the category, status, crime etc can go in the notes property

Comment (jbothma) [inline datasets/us/ice_wanted/crawler.py:31]:
```suggestion
- Crime details: {kwargs.get('crime_details')}
```

**Review by jbothma — COMMENTED** [review]
very nice, thanks! Just some tweaks then I think it's ready

File: datasets/us/ice_wanted/crawler.py (hunk 3 above)
Comment (atehe) [reply datasets/us/ice_wanted/crawler.py:55]: You're right.
Some of them had two names before the comma, eg https://www.ice.gov/most-wanted/garcia-jerez-anibal,  so I though it was an alias

File: datasets/us/ice_wanted/crawler.py (hunk 2 above)
Comment (atehe) [reply datasets/us/ice_wanted/crawler.py:31]: Got it

**Pull request merged**